src/Columns/ColumnIndex.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <Columns/IColumn.h>
+#include <Columns/ColumnsNumber.h>
+
+namespace DB
+{
+
+/// Wrapper around ColumnVector to store indexes.
```

Comment:
I think this part was ejected from LowCardinality with no change, so I can skip reviewing it.

Reply:
Yes, exactly